README.md

```diff
@@ -1,49 +1,78 @@
-> https://www.researchgate.net/project/Resolving-Known-Plaintext-Attack-on-FTIE-RT-ACM
-
-## Research Proposal Abstract
-[File to Image Encryption (FTIE-RT-ACM)](https://www.researchgate.net/publication/320087595_File_To_Image_Encryption_FTIE_Menggunakan_Algoritma_Randomized_Text_Dan_Arnold_Cat_Map_ACM_Untuk_Keamanan_Transmisi_Data_Digital) technique that makes use of
-[Randomized Text algorithm (RT)](https://s3.amazonaws.com/academia.edu.documents/46998494/Randomized_Text_Encryption_a_New_Dimensi20160704-24345-9laysm.pdf?AWSAccessKeyId=AKIAIWOWYYGZ2Y53UL3A&Expires=1515661445&Signature=GGievOHY%2FRvQcBoILP5nWSizts0%3D&response-content-disposition=inline%3B%20filename%3DRandomized_Text_Encryption_a_New_Dimensi.pdf)
-and [Arnold Cat Map algorithm (ACM)](https://en.wikipedia.org/wiki/Arnold%27s_cat_map)
-have a weakness in RT algorithm block and ACM algorithm block.
-Known plaintext attack that exploits a simple decryption equation
-in RT algorithm block can be applied to generate the key used in the encryption.
-This is could be done by FTIE-RT-ACM technique specifically in algorithm RT block
-that does not implement a keystream generator, that in term of encryption it is useful
-to keep the key safe.
-Furthermore, the use of ACM 1D algorithm in ACM algorithm block has no proof of correctness.
-
-This research proposes FTIE-RT-ACM technique implement keystream generator in
-RT algorithm block and ACM 2D in the ACM algorithm block.
-The keystream generator that will be used is [Blum Blum Shub (BBS) keystream generator algorithm](https://en.wikipedia.org/wiki/Blum_Blum_Shub).
-BBS proved that it is not easy to predict given polynomial time.
-Furthermore, the use of ACM 2D algorithm instead of ACM 1D algorithm gives more guarantee of
-the algorithm correctness.
-
-## Research Proposal in Indonesian
-For you who willing to know more about my research, I have already done the research proposal available in Indonesia only (sad).
-Not an Indonesian, just drop a request and I will work on it :D.
-The research proposal then available by contacting me (DM, tweet, mention or anything) via Twitter: [@SatriaHRHarsono](https://twitter.com/SatriaHRHarsono).
-
-## System Block Diagram
+
 ![System Block Diagram](docs/assets/system-block-diagram.png)
+Project description
+
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+Python 3.6
+Virtualenv
+```
+pip install virtualenv
+```
+
+### Installing
+
+Steps to follow:
+1. Clone this repo from master branch https://github.com/hafizhme/ftie-rt-acm
+2. From the root project directory, Create a directory for virtual environment
+3. Create virtual environment by `virtualenv your-env`
```

Comment:
Please add a consideration for `virtualenv`'s naming. Some name will be ignored by git, see https://github.com/hafizhme/ftie-rt-acm/blob/master/.gitignore#L85